pages/demo/index.tsx

```diff
@@ -22,13 +21,14 @@ function Demo() {
     setIiifContent(tempIiifContent)
   };
 
-  const options = {
-    showIIIFBadge: false,
+  const options: ViewerConfigOptions = {
+    ignoreCaptionLabels: ["Clips"],
```

Comment:
@wykhuh Looks like Mat Jordan already thought of this 😀. I tested this out in Mac Brave/Chrome/Safari and "Clips" are hidden as expected.